src/core/Akka/Actor/SupervisorStrategy.cs

```diff
@@ -349,8 +349,9 @@ public OneForOneStrategy(int maxNrOfRetries, int withinTimeMilliseconds, IDecide
         /// <summary>
         /// Constructor that accepts only a decider and uses reasonable defaults for the other settings
         /// </summary>
-        /// <param name="localOnlyDecider">TBD</param>
-        public OneForOneStrategy(Func<Exception, Directive> localOnlyDecider) : this(-1, -1, localOnlyDecider, true)
+        /// <param name="localOnlyDecider">mapping from Exception to <see cref="Directive" /></param>
+        /// <param name="loggingEnabled">If <c>true</c> failures will be logged</param>
+        public OneForOneStrategy(Func<Exception, Directive> localOnlyDecider, bool loggingEnabled = true) : this(-1, -1, localOnlyDecider, loggingEnabled)
```

Comment:
Need to change this. Can't change an existing public API signature. Add a new overload instead.